src/sinks/influxdb/metrics.rs

```diff
@@ -144,6 +146,7 @@ impl Service<Vec<Metric>> for InfluxDBSvc {
             self.protocol_version,
             items,
             self.config.namespace.as_deref(),
+            self.config.tags.clone(),
```

Comment:
I *imagine* you can make this work without requiring a clone here. Below, on l180, you do another clone after an `.into_iter()`. I wonder if you can make this a borrow and that inner clone will be enough?